docs/book/src/tasks/upgrading-clusters.md

```diff
@@ -47,6 +47,28 @@ that if a specific machine image is specified, it has to match the Kubernetes ve
 `KubeadmControlPlane` spec. In order to only trigger a single upgrade, the new `MachineTemplate` should be created first
 and then both the `Version` and `InfrastructureTemplate` should be modified in a single transaction.
 
+#### How to schedule a machine rollout
+
+A `KubeadmControlPlane` resource has a field `UpgradeAfter` that can be set to a timestamp
```

Comment:
Sidenote: We should rename this field to RolloutAfter in v0.4.0, which is a ⚠️ breaking change

Reply:
@shysank Would you be interested in ^?

Reply:
how urgent is this? I'm kind of occupied with other things right now.